crates/pink/sidevm/env/src/lib.rs

```diff
@@ -0,0 +1,627 @@
+    Reserved203 = 203,
+    /// Reserved for future use
+    Reserved204 = 204,
+    /// Reserved for future use
+    Reserved205 = 205,
+    /// Reserved for future use
+    Reserved206 = 206,
+    /// Reserved for future use
+    Reserved207 = 207,
+    /// Reserved for future use
+    Reserved208 = 208,
+    /// Reserved for future use
+    Reserved209 = 209,
+    /// Reserved for future use
+    Reserved210 = 210,
+    /// Reserved for future use
+    Reserved211 = 211,
+    /// Reserved for future use
+    Reserved212 = 212,
+    /// Reserved for future use
+    Reserved213 = 213,
+    /// Reserved for future use
+    Reserved214 = 214,
+    /// Reserved for future use
+    Reserved215 = 215,
+    /// Reserved for future use
+    Reserved216 = 216,
+    /// Reserved for future use
+    Reserved217 = 217,
+    /// Reserved for future use
+    Reserved218 = 218,
+    /// Reserved for future use
+    Reserved219 = 219,
+    /// Reserved for future use
+    Reserved220 = 220,
+    /// Reserved for future use
+    Reserved221 = 221,
+    /// Reserved for future use
+    Reserved222 = 222,
+    /// Reserved for future use
+    Reserved223 = 223,
+    /// Reserved for future use
+    Reserved224 = 224,
+    /// Reserved for future use
+    Reserved225 = 225,
+    /// Reserved for future use
+    Reserved226 = 226,
+    /// Reserved for future use
+    Reserved227 = 227,
+    /// Reserved for future use
+    Reserved228 = 228,
+    /// Reserved for future use
+    Reserved229 = 229,
+    /// Reserved for future use
+    Reserved230 = 230,
+    /// Reserved for future use
+    Reserved231 = 231,
+    /// Reserved for future use
+    Reserved232 = 232,
+    /// Reserved for future use
+    Reserved233 = 233,
+    /// Reserved for future use
+    Reserved234 = 234,
+    /// Reserved for future use
+    Reserved235 = 235,
+    /// Reserved for future use
+    Reserved236 = 236,
+    /// Reserved for future use
+    Reserved237 = 237,
+    /// Reserved for future use
+    Reserved238 = 238,
+    /// Reserved for future use
+    Reserved239 = 239,
+    /// Reserved for future use
+    Reserved240 = 240,
+    /// Reserved for future use
+    Reserved241 = 241,
+    /// Reserved for future use
+    Reserved242 = 242,
+    /// Reserved for future use
+    Reserved243 = 243,
+    /// Reserved for future use
+    Reserved244 = 244,
+    /// Reserved for future use
+    Reserved245 = 245,
+    /// Reserved for future use
+    Reserved246 = 246,
+    /// Reserved for future use
+    Reserved247 = 247,
+    /// Reserved for future use
+    Reserved248 = 248,
+    /// Reserved for future use
+    Reserved249 = 249,
+    /// Reserved for future use
+    Reserved250 = 250,
+    /// Reserved for future use
+    Reserved251 = 251,
+    /// Reserved for future use
+    Reserved252 = 252,
+    /// Reserved for future use
+    Reserved253 = 253,
+    /// Reserved for future use
+    Reserved254 = 254,
+    /// Reserved for future use
+    Reserved255 = 255,
```

Comment:
Adding all these variants is to prevent breaking the already deployed guest program if we add new defined variants in the future.

Reply:
I still didn't understand the reservation here. When do you think this is relevant?

Reply:
When we add a new error code and it is returned by an exsisting API, the guest program would be unable to decode the new variant and crash without reserving it in advance since it was compiled from an old `enum Error` definition.

Reply:
Oh, got it